plugins/hlf.yaml

```diff
@@ -3,13 +3,12 @@ kind: Plugin
 metadata:
   name: hlf
 spec:
-  version: "v1.1.0"
+  version: "v1.2.1"
   homepage: https://github.com/kfsoftware/hlf-operator
-  shortDescription: "Deploy and manage Hyperledger Fabric components"
+  shortDescription: "Deploy and manage Hyperledger Fabric Peers, Orderers and Certificate authorities"
```

Comment:
This needs to be short and fit to krew list

Reply:
Solved